.gitignore

```diff
@@ -47,8 +47,8 @@ coverage.xml
 *.mo
 *.pot
 
-# Django stuff:
-*.log
+# System stuff
+.DS_Store
```

Comment:
This is a system config and should be treated as such – meaning in the `~/.gitignore` – not in the project.